spynnaker/pyNN/models/neuron/population_machine_vertex.py

```diff
@@ -130,8 +144,9 @@ def get_minimum_buffer_sdram_usage(self):
 
     @overrides(AbstractReceiveBuffersToHost.get_n_timesteps_in_buffer_space)
     def get_n_timesteps_in_buffer_space(self, buffer_space, machine_time_step):
+        safe_space = buffer_space - self._overflow_sdram
```

Comment:
If get_buffered_sdram_per_timestep is changed to get_n_timesteps_in_buffer_space, this can now call neuron_recorder.get_n_timesteps_in_buffer_space.

Reply:
See above on why get_n_timesteps_in_buffer_space is bad!